docs/user-guide/testing/load_tests.md

```diff
@@ -0,0 +1,64 @@
+| true      | S1          | 1             | 140               | 4                         | 10             | 2940                | 0                      | 500                            |
+| true      | S3          | 1             | 900               | 4                         | 15             | 5400                | 0                      | 500                            |
+| false     | S3          | 1             | 1100              | 4                         | -              | -                   | -                      | -                              |
+| true      | S3          | 2             | 300               | 2                         | 9              | 4800                | 0                      | 600                            |
+| false     | S3          | 2             | 900               | -                         | -              | -                   | -                      | -                              |
+
+Issues encountered in these load tests:
+
+- [Unexpected ObjectDisposedException when IoT Hub is throttling · Issue #6042 · Azure/iotedge (github.com)](https://github.com/Azure/iotedge/issues/6042)
+- When executing the load tests for more than 1000 devices, we started to see many `AmqpException` with the message `Unable to open Events link as the connection could not be authenticated`
+- In case of two gateways and high message rates (10 messages per second), we saw that the delivery latency spiked for one of the two gateways. This behavior did not occur with lower message rates (3 messages per second). This correlated with the IoT Edge hub queue filling up, and as such it's likely that this caused the delivery latency spikes.
+
+The memory/CPU usage of the IoT Edge host in gateway mode (for 900 devices) was fairly stable at around 1GB of memory used and an average CPU consumption of 40% on a Standard D2s v3 Debian 11 VM:
+
+![gateway-mode-host-cpu-memory](..\..\images\lt-host-gateway-mode.png)
+
+### January 21st 2022, load test of v2.0.0-beta1 in direct mode (ENABLE_GATEWAY=false)
+
+This load test used the default deduplication strategy. First, we deployed two LNS on Standard D2s v3 Debian 11 VMs (2vCPU, 8GB of memory). We simulated 1600 OTAA devices, distributed among 8 factories with 4 concentrators each. All devices are in reach of all concentrators within the same factory and send 5 messages each, starting with a join rate of 1.5 messages per second (to not hit the IoT Hub S1 quota) and then with progressively sending messages faster and faster (starting at 1.5 messages per second to pre-warm the cache and not hit IoT Hub S1 quota, and then successively increase the load to around 9.5 messages per second). Keep in mind that the effective message rate is higher than 9.5 messages per second, since every message is delivered to the gateways by the four concentrators per factory. The following bugs/issues became apparent in the load test:
```

Comment:
```suggestion
This load test used the default deduplication strategy. First, we deployed two LNS on Standard D2s v3 Debian 11 VMs (2vCPU, 8GB of memory). We simulated 1600 OTAA devices, distributed among 8 factories with 4 concentrators each. All devices are in reach of all concentrators within the same factory and send 5 messages each, starting with a join rate of 1.5 messages per second and then progressively sending messages faster and faster (starting at 1.5 messages per second to pre-warm the cache and not hit IoT Hub S1 quota, and then successively increase the load to around 9.5 messages per second). Keep in mind that the effective message rate is higher than 9.5 messages per second, since every message is delivered to the gateways by the four concentrators per factory. The following bugs/issues became apparent in the load test:
```